docs/index.rst

```diff
@@ -29,6 +30,13 @@ Welcome to DeepForge's documentation!
    deployment/overview.rst
    deployment/native.rst
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Example Projects
```

Comment:
It may be best to use both ("Tutorials and Example Projects") given that the redshift project is not, strictly speaking, a tutorial. It would also give us a space to add links and documentation on other example projects down the line. Though, if numerous other examples are added, it might necessitate separating Tutorials and Example Projects into separate sections.